### PR TITLE
Add Interlock ASG

### PR DESCRIPTION
**Pull request by ericonr**

### CODE REVIEW

File: iocBoot/iocBPMRFFEApp/run.sh [hunk 1]
```diff
@@ -14,6 +14,8 @@ device_prefix_var=CRATE_${CRATE_NUMBER}_BPM_${BPM_NUMBER}_PV_DEVICE_PREFIX
 export EPICS_PV_AREA_PREFIX=${!area_prefix_var}
 export EPICS_PV_DEVICE_PREFIX=${!device_prefix_var}
 
+export R_BPM=${EPICS_PV_DEVICE_PREFIX%RFFE}
+
```
Comment (henriquesimoes) [inline iocBoot/iocBPMRFFEApp/run.sh]: `EPICS_PV_DEVICE_PREFIX` is already what you want. We append RFFE in the definition of `R`.

We can rename it to `EPICS_PV_BPM_PREFIX` or something shorter to make this clearer.

File: iocBoot/iocBPMRFFEApp/st.cmd [hunk 2]
```diff
@@ -14,6 +14,9 @@ epicsEnvSet("R", "${EPICS_PV_DEVICE_PREFIX}RFFE")
 epicsEnvSet("STREAM_PROTOCOL_PATH", "$(TOP)/BPMRFFEApp/Db")
 epicsEnvSet("PORT", "BPMRFFE")
 
+asSetFilename("$(TOP)/BPMRFFEApp/Db/accessSecurityFile.acf")
```
Comment (henriquesimoes) [inline iocBoot/iocBPMRFFEApp/st.cmd]: Should that be installed? I've never quite understood when we install databases and other related configuration and when we do not.

IMHO, we should be able to `make install` to another location and still be able to run it successfully.

**Review by henriquesimoes — CHANGES_REQUESTED** [review]

File: iocBoot/iocBPMRFFEApp/run.sh (hunk 1 above)
Comment (ericonr) [reply iocBoot/iocBPMRFFEApp/run.sh]: Nice catch, thanks. It hadn't worked correctly when I used `R`, but I didn't check where we added `RFFE`.

File: iocBoot/iocBPMRFFEApp/st.cmd (hunk 2 above)
Comment (ericonr) [reply iocBoot/iocBPMRFFEApp/st.cmd]: > IMHO, we should be able to make install to another location and still be able to run it successfully.

Agreed, but I don't know how common/supported that actually is. Most `.db` or `.templates` are installed, but `.proto` files often aren't, and I don't know about `.acf` ones. And even if you look at something like autosave and asyn: the template for autosave records isn't installed, the one for the asyn record is.

I will check if the build system takes `.acf` files in the install list.

**Review by henriquesimoes — APPROVED** [review]
Looks nice!

Just a note about updating epics-in-docker for v0.4.0: we should later think about making it possible to use `lnls-run` in the use case we have here.

**Pull request merged**